packages/pong-engine/src/queue.dto.ts

```diff
@@ -17,8 +23,8 @@ export enum GameStatus {
 }
 
 export type GameChallengeResponseDto = {
```

Comment:
why do we need classes with their constructors? This appears to work.

Reply:
I think I will leave this for some ticket I'll open now on the topic of sharing dtos between webapp and transcendence-app for websockets events. The ability of using the `ValidationPipe` decorator from class-validator does not seem to me to add so much value, here and now.

Reply:
Opened ticket [here](https://github.com/42AGV/ft_transcendence/issues/476).